ibis/expr/tests/test_value_exprs.py

```diff
@@ -50,26 +50,24 @@ def test_unicode():
     assert False
 
 
-@pytest.mark.parametrize(
-    ['value', 'expected_type'],
-    [
-        (5, 'int8'),
-        (127, 'int8'),
-        (128, 'int16'),
-        (32767, 'int16'),
-        (32768, 'int32'),
-        (2147483647, 'int32'),
-        (2147483648, 'int64'),
-        (-5, 'int8'),
-        (-128, 'int8'),
-        (-129, 'int16'),
-        (-32769, 'int32'),
-        (-2147483649, 'int64'),
-        (1.5, 'double'),
-        ('foo', 'string'),
-    ]
-)
-def test_literal_cases(value, expected_type):
+@pytest.mark.parametrize(['value', 'expected_type'], [
```

Comment:
Can you leave this formatting as is?

Reply:
OK!

Reply:
for some unknown reason I lost this ... I am doing that.